transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -827,11 +827,14 @@ private void write(Object msg, boolean flush, ChannelPromise promise) {
                 next.invokeWrite(m, promise);
             }
         } else {
+            boolean wakeup = Boolean.TRUE.equals(channel().config().getOption(ChannelOption.WAKEUP_ON_WRITE));
```

Comment:
@Scottmitch I was thinking about this but thought it is not worth it as this only happens for writes from outside the EventLoop which are "expensive" anyway.